Gemfile.lock

```diff
@@ -198,16 +198,6 @@ PATH
     veteran_confirmation (0.0.1)
     veteran_verification (0.0.1)
 
-GEM
```

Comment:
Don't commit these changes to the lock file.

Reply:
Herbert, it looks like this file hasnt been uncommitted, are you able to revert this commit?